app/LogFile.php

```diff
@@ -49,14 +49,14 @@ public static function put(string $folder, array $contents): ?string
         // Make sure folder ends with a /
         $folder = Str::finish($folder, '/');
 
-        $filename = Carbon::now()->format(self::NAME_FORMAT);
+        $filename = Carbon::now()->format(static::NAME_FORMAT);
```

Comment:
Now that we're extending this class, it becomes obvious that none of these props or methods should have used `self::` they should have used `static::` so children get their own implementations not the parent's.